File: usedesk_sdk/src/main/java/ru/usedesk/sdk/data/framework/api/standard/entity/response/SendFeedbackResponse.java
package ru.usedesk.sdk.data.framework.api.standard.entity.response;

public class SendFeedbackResponse extends BaseResponse {

    public static final String TYPE = "@@chat/current/CALLBACK_ANSWER";

    private Answer answer;

    public SendFeedbackResponse() {
        super(TYPE);
    }

    public Answer getAnswer() {
        return answer;
    }

    private class Answer {

        private boolean status;

        private boolean isStatus() {
            return status;
        }
    }
}

File: chat-sdk/src/main/java/ru/usedesk/chat_sdk/internal/data/framework/socket/entity/response/SendFeedbackResponse.kt
package ru.usedesk.chat_sdk.internal.data.framework.socket.entity.response

class SendFeedbackResponse : BaseResponse(TYPE) {
    val answer: Answer? = null

    class Answer {
        private val isStatus = false
    }

    companion object {
        const val TYPE = "@@chat/current/CALLBACK_ANSWER"
    }
}